bigquery/google/cloud/bigquery/magics.py

```diff
@@ -324,6 +324,15 @@ def _run_query(client, query, job_config=None):
         "the context default_query_job_config.maximum_bytes_billed."
     ),
 )
+@magic_arguments.argument(
+    "--dry_run",
+    action="store_true",
+    default=False,
+    help=(
+        "Sets query to be a dry run to estimate costs"
```

Comment:
A period and a space are missing at the end of the sentence, looks strange in the help output.

```suggestion
        "Sets query to be a dry run to estimate costs. "
```